config/tests/samples/create/harness.go

```diff
@@ -236,6 +240,8 @@ func NewHarness(ctx context.Context, t *testing.T) *Harness {
 		kccConfig.GCPAccessToken = h.gcpAccessToken
 	} else if targetGCP := os.Getenv("E2E_GCP_TARGET"); targetGCP == "real" {
 		t.Logf("targeting real GCP")
+	} else if targetGCP := os.Getenv("E2E_GCP_TARGET"); targetGCP == "vcr" {
```

Comment:
Non-blocking future improvement idea: Can we have all the environment variables and valid values used by integration tests documented and maintained somewhere? As someone who rarely touches the code here, it's hard for me to figure out what environment variables I should set and why.

Reply:
I created b/332719149 to address this comment.